docs/source/tutorials/one_class_segmentation_2d_unet.rst

```diff
@@ -222,23 +433,33 @@ Evaluate model
 
     To test the trained model on the testing sub-dataset and compute evaluation metrics, run:
 
-    .. code-block:: bash
+    .. tabs::
+
+        .. tab:: Command Line Interface
+
+            .. code-block:: bash
+
+               ivadomed --test -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
 
-       ivadomed --test -c config.json --path-data path/to/bids/data --path-output path/to/output/directory
+        .. tab:: JSON File
 
-    If you prefer to use config files over CLI flags, set ``command`` to the following in you config file:
+            If you prefer to use config files over CLI flags, set ``command`` to the following in you config file:
 
-    .. code-block:: json
+            .. code-block:: json
 
-       "command": "test"
+               "command": "test"
 
-    You can also set ``path_output``, and ``path_data`` arguments in your config file.
+            You can also set ``path_output``, and ``path_data`` arguments in your config file, usually around **Line Number 2, 4, and 12** in the `config.json` respectively.
```

Comment:
We shouldn't refer to specific line numbers here.
I don't think we need to add a link either, as those argument are widely discuss beforehand in the tutorial.

Reply:
Good point. I agree.